backend/s2s_server_pipeline.py

```diff
@@ -25,8 +25,13 @@
 from funasr import AutoModel
 # TTS
 import torchaudio
-from cosyvoice.utils.file_utils import load_wav
-from cosyvoice.cli.cosyvoice import CosyVoice
+try:
+    from cosyvoice.utils.file_utils import load_wav
+    from cosyvoice.cli.cosyvoice import CosyVoice
+except ImportError as e:
+    print(f"Error: {e}")
```

Comment:
print(f"Failed to import CosyVoice modules. Please ensure cosyvoice is installed (pip install cosyvoice). Error: {e}")